v2_papers/2009/kind_histogram.json

```diff
@@ -0,0 +1,179 @@
+      {
+        "name": "data center",
+        "types": [],
+        "generation_year": 2007,
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "covered_period": "months"
+      },
+      {
+        "name": "DARPA IDeval",
+        "availability": "public",
+        "format": "packet",
+        "types": [],
+        "generation": "altered_captured",
+        "generation_year": 1999,
+        "covered_period": "weeks",
+        "details": [
+          "raw"
+        ],
+        "subsets": []
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": true,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [],
+    "normalization_type": "no",
+    "transformations": [
+      "_histograms",
+      "_metric_space",
+      "entropy"
+    ],
+    "final_data_format": "_histograms",
+    "feature_selections": [
+      {
+        "name": "constant-removal",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      },
+      {
+        "name": "pca",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      }
+    ],
+    "flows": [
+      {
+        "selection": "study_based",
+        "role": "main",
+        "main_goal": "anomaly_detection",
+        "active_timeout": "missing",
+        "idle_timeout": "missing",
+        "features": [
+          "sourceIPv4Address",
+          "destinationIPv4Address",
+          "sourceTransportPort",
+          "destinationTransportPort",
+          "protocolIdentifier",
+          "octetTotalCount",
+          "packetTotalCount",
+          "flowDurationMilliseconds"
+        ],
+        "key_features": []
+      }
+    ]
+  },
+  "analysis_method": {
+    "supervised_learning": true,
+    "unsupervised_learning": true,
+    "semisupervised_learning": false,
+    "anomaly_detection": true,
+    "tools": [],
+    "algorithms": [
+      {
+        "name": "hierarchical cluster analysis",
+        "subname": "hca",
+        "learning": "unsupervised",
+        "role": "intermediate",
+        "type": "clustering",
+        "metric/decision_criteria": "euclidean",
+        "source": "referenced",
+        "parameters_provided": false
+      },
+      {
+        "name": "clustering",
+        "subname": "K-means",
+        "learning": "unsupervised",
+        "role": "intermediate",
+        "type": "clustering",
+        "metric/decision_criteria": "euclidean",
+        "source": "referenced",
+        "parameters_provided": false
+      }
+    ]
+  },
+  "evaluation": {
+    "algorithm_comparison": false,
+    "internal_validation": true,
+    "external_validation": true,
+    "dpi-based_validation": false,
+    "port-based_validation": false,
+    "pre-knowledge-based_validation": false,
```

Comment:
At least for the IDEval dataset they used ``pre-knowledge-based_validation``. I'm not sure how they got the labels for the other datasets though.

Reply:
What do yo suggest? (for IDEval they had labels only for flows and not for histograms)

Reply:
Although I didn't explicitly read this, my guess is that they label histograms as anomalous if there is an anomalous flow in them, correct? If so, I'd say they use ``pre-knowledge-based_validation`` for the IDEval.

Reply:
but for the other data-sets they didn't mention anything, I'll keep it like this for now